src/components/MoviesPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { MovieResults } from "../utils/types";
+import { Box, Text } from "@chakra-ui/react";
+import Movie from "./Movie";
+
+export const MoviesPage = ({ loadedMovie }: { loadedMovie: MovieResults }) => {
```

Comment:
Niech MoviesPage dostaje loadedMovie i renderuje, a komponent wyżej sprawdza czy to może być undefined i jeśli jest to nie używa tego komponentu